### PR TITLE
Don't force-show the watermark

### LINKED ISSUE #24: Watermark
Remove the forced watermark mode

### PR DESCRIPTION
**Pull request by Natan515**
fix #24

### CODE REVIEW

**Review by huddy987 — CHANGES_REQUESTED** [review]
On line 19 you will need to update closeable so that non-premium users can toggle the watermark

**Comment by Natan515** [comment]
Oh, sorry I though it was just changing true to false lol. Will do

**Review by huddy987 — DISMISSED** [review]
This looks good, but there is still some strange behavior with the watermark that needs to be investigate (when HUD is disabled if watermark is enabled it is still rendered). I will look into this.

**Comment by Natan515** [comment]
It seems good to me

**Comment by Natan515** [comment]
I just tried it and it works great

File: src/main/kotlin/com/lambda/client/gui/hudgui/elements/client/WaterMark.kt [hunk 1]
```diff
@@ -2,22 +2,19 @@ package com.lambda.client.gui.hudgui.elements.client
...
-import com.lambda.client.module.modules.client.Capes
 import com.lambda.client.util.graphics.VertexHelper
 import org.lwjgl.opengl.GL11.glScalef
 
 internal object WaterMark : LabelHud(
     name = "Watermark",
     category = Category.CLIENT,
     description = "Lambda watermark",
-    enabledByDefault = true
+    enabledByDefault = false
```
Comment (Avanatiker) [inline src/main/kotlin/com/lambda/client/gui/hudgui/elements/client/WaterMark.kt]: Should be enabled at least. If player wants he can disable

Comment (huddy987) [inline src/main/kotlin/com/lambda/client/gui/hudgui/elements/client/WaterMark.kt]: We should enable the watermark by default

**Review by huddy987 — CHANGES_REQUESTED** [review]

**Review by huddy987 — APPROVED** [review]

**Comment by Natan515** [comment]
Dude no! That's the whole thing, that it will be false of default... U want this to be on default but discord rpc no...

**Comment by Natan515** [comment]
Pls undo what u did

**Comment by Avanatiker** [comment]
Works like intended now.

**Comment by Avanatiker** [comment]
Also @Natan515 pls sign ur commits with GPG in future.

**Comment by Avanatiker** [comment]
Also can u plase make a new branch for patches and keep master clean? It messes with my git becuase of the similar names

**Review by Avanatiker — APPROVED** [review]

**Pull request merged**